pkgs/applications/misc/megasync/default.nix

```diff
@@ -98,7 +95,7 @@ mkDerivation rec {
     "--with-cryptopp"
     "--with-curl"
     "--with-ffmpeg"
-    "--with-freeimage"
+    "--without-freeimage"
```

Comment:
Is this necessary? seems like disabling freeimage has to be done in too many places :thinking:

Reply:
It looks to be necessary. Without the line the compilation fails like this:

```
$ nix build --nix-path nixpkgs=/home/michojel/.cache/nixpkgs-review/rev-b3b106b7a6d0d9d1300b54a4fe3ff4f98179afb1/nixpkgs nixpkgs-overlays=/tmp/tmp9zpa397q --extra-experimental-features nix-command no-url-literals --no-link --keep-going --no-allow-import-from-derivation --option build-use-sandbox relaxed -f /home/michojel/.cache/nixpkgs-review/rev-b3b106b7a6d0d9d1300b54a4fe3ff4f98179afb1/build.nix
error: builder for '/nix/store/mdkzammq8z6m792i1al65kxpyqc95ymg-megasync-4.9.1.0.drv' failed with exit code 1;
       last 10 log lines:
       > checking for SSL_new in -lssl... yes
       > checking for c-ares... checking for ares.h... yes
       > checking for ares_library_init in -lcares... yes
       > checking for curl/curl.h... yes
       > checking for main in -lcurl... yes
       > checking whether cURL is built with OpenSSL support... yes
       > checking if building example applications... no
       > checking for FreeImage... --with-freeimage not specified
       > checking for FreeImage.h... no
       > configure: error: FreeImage.h header not found or not usable
       For full logs, run 'nix log /nix/store/mdkzammq8z6m792i1al65kxpyqc95ymg-megasync-4.9.1.0.drv'.
error: 1 dependencies of derivation '/nix/store/xgazyqgns7v6dl9ywy7fz83nxvfnwa04-review-shell.drv' failed to build
1 package failed to build:
megasync
```